File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// PlaystationStoreApi2 SDK exists test

require_once __DIR__ . '/../playstationstoreapi2_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = PlaystationStoreApi2SDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# PlaystationStoreApi2 SDK exists test

require "minitest/autorun"
require_relative "../PlaystationStoreApi2_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PlaystationStoreApi2SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
